tools/topology/sof-hda-generic.m4

```diff
@@ -24,6 +24,8 @@ ifelse(CHANNELS, `0', ,
 define(DMIC_PIPELINE_48k_ID, `10')
 define(DMIC_PIPELINE_16k_ID, `11')
 
+DEBUG_START
```

Comment:
is this supposed to be here?

Reply:
I'm not sure if there's any overhead of having this. I'll check if it impact *.tplg files. With DEBUG_START - DEBUG_END we are able to plot the topology as described in

https://thesofproject.github.io/latest/developer_guides/topology/topology.html#debugging-topology

Some topologies have them and some do not.

Reply:
But this fix/enhance does not depend on the DEBUGs so I think I should remove them from this PR.

Reply:
the DEBUG should not propagate to .conf or .tplg file if you are just using the m4 and alsa tools as defined in makefiles. As you said it is just a helper to plot the graph. And you can just remove it as it is not part of this PR. Anyway, good to spread the info how to plot the topology :)

Reply:
Done!